config/locales/pt-BR.yml

```diff
@@ -91,6 +91,8 @@ pt-BR:
     tasks: "Tarefas"
     notes: "Notas"
     attachments: "Anexos"
+    warnings:
+      backlogged_release: "Esse lançamento está com risco de ser completo depois da data de lançamento definida."
```

Comment:
Special attention to this translation, it might (and probably is) wrong, grammatically.